src/mainwindow.ui

```diff
@@ -465,6 +471,175 @@
              </widget>
             </item>
             <item>
+             <widget class="QGroupBox" name="systemInfoGroupBox">
+              <property name="title">
+               <string>System Information</string>
+              </property>
+              <property name="checkable">
+               <bool>true</bool>
+              </property>
+              <property name="checked">
+               <bool>false</bool>
+              </property>
+              <layout class="QFormLayout" name="formLayout_2">
+               <item row="0" column="0">
+                <widget class="QLabel" name="hostNameLabel">
```

Comment:
I just realize that none of the labels on the summary page are selectable. Can you add that feature (afaik it's a property on the label), such that we can copy'n'paste from the summary page?